docs/administration/retention-policies/index.md

```diff
@@ -65,17 +80,48 @@ You can keep all, or select a number of releases to keep.
 
 You are also able to specify a number of days worth of releases and files to keep if this is preferred.
 
-## Built-in feed retention policy {#RetentionPolicies-builtinfeed-retentionpolicy}
+## Set Built-in feed retention policy {#set-builtinfeed-retentionpolicy}
 
 You can find the built-in repository retention policy settings under **{{Library,Packages}}**.
 
 ![](images/3278060.png "width=500")
 
-This can also be set to keep a set number, or keep for a set number of days:
+This can also be set to keep packages forever, or for a set number of days:
```

Comment:
Should there be a caveat here for Cloud?

Reply:
Agreed, hint added (and to the release section too :smile:)